server/src/main/java/org/elasticsearch/index/query/DistanceFeatureQueryBuilder.java

```diff
@@ -120,6 +120,7 @@ protected Query doToQuery(QueryShardContext context) throws IOException {
             return Queries.newMatchNoDocsQuery("Can't run [" + NAME + "] query on unmapped fields!");
         }
         Object originObj = origin.origin();
+        // TODO these ain't gonna work with runtime fields
```

Comment:
interesting too, at least both of these are around parsing values, which you are making public so that we can reuse for runtime fields?

Reply:
Yeah, we can handle this *somehow*.